js/templates/settings.html

```diff
@@ -1261,6 +1261,35 @@ <h3 class="padding15 margin0 fontWeight500"><%= polyglot.t('Advanced') %></h3>
               </div>
 
             </div>
+            <div class="flexRow">
+
+              <div class="flexCol-3  borderRight custCol-border">
+                <div class="fieldItem">
+                  <label>
+                     Connected Peers
+                  </label>
+                </div>
+              </div>
+
+              <div class="flexCol-9 borderRight0 custCol-border">
+                <div class="fieldItem flexRow">
+                  <div class="flexRow">
+                    <div class="flexCol-9">
+                      <span class="js-numConnectedPeers"></span>
+                    </div>
+                    <div class="positionTopRight paddingright5">
+                      <a id="showPeersButton" class="btn btn-txt js-showPeers custCol-secondary">
+                        Show Connected Peers
+                      </a>
+                    </div>
+                  </div>
+                  <div class="flexRow">
+                    <div class="peersList js-connectedPeers"></div>
```

Comment:
This div needs the custCol-text class, otherwise if someone sets a theme color the text becomes the same color as the background and is invisible. 

You can try it by setting colors in the Page tab.